### DISCUSSION TITLE
role/user impersonation for security testing

### DISCUSSION THREAD

**Discussion by cosminioan** [DaxStudio/DaxStudio#682] [General]
hi folks, I was wondering if there is a way to connect with Role/User impersonation for security testing.
It appears that there is Effective User Name (domain\user)  option in Connection but that does not seem to be working (it seems to deny all access, even though via SSMS, or Excel, the respective user has access to the data).

thank you kindly for any feedback.
Cos

**Comment by cosminioan** [comment]
please disregard, Effective User Name login looks OK. (not able to delete/close ticket.

**Comment by dgosbell** [comment]
Yes, this is just a discussion, not an issue/ticket, so it does not have a close option. Glad to hear you got this working
